src/components/SubmitResponse/SubmitResponse.js

```diff
@@ -0,0 +1,208 @@
+import { useState, useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import {
+    Box,
+    Text,
+    Input,
+    Textarea,
+    Button,
+    Flex,
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    useDisclosure,
+} from '@chakra-ui/react';
+
+import { useGetQuestionsContext } from '../../context/GetQuestionsContext';
+import GetCompanyQuestionsService from '../../services/GetCompanyQuestionsService';
+import { FaCloudUploadAlt } from 'react-icons/fa';
+const SubmitResponse = () => {
+    const [pageNumber, setPageNumber] = useState(1);
+    const [arrayOfAnswers, setArrayOfAnswers] = useState([]);
+    const [finishedAnswers, setFinishedAnswers] = useState(false);
+    const { isOpen, onOpen, onClose } = useDisclosure();
+
+    const { companyQuestions, getCompanyQ } = useGetQuestionsContext();
+
+    const {
+        register,
+        handleSubmit,
+
+        resetField,
+        formState: { errors },
+    } = useForm({
+        defaultValues: {
+            answer: '',
+        },
+    });
+
+    let companyId;
+    let userData = JSON.parse(localStorage.getItem('userData'));
+    let questionsPerPage = 1;
+    let pagesVisited = questionsPerPage * pageNumber;
+    let totalPages = companyQuestions.length;
+
+    useEffect(() => {
+        if (userData && arrayOfAnswers.length == 0) {
+            companyId = userData.username;
```

Comment:
@hristinakovacevic companyId should be read from user's profile

And the get the user's profile we need to call the api: `/api/profiles?filter[user][id][$eq]=${user.id}` :)
Then we can GET `/api/companies/${profile.company}` (or you can use `populate=company` on the profile request to get company info

Reply:
Done.